base_ebill_payment_contract/tests/test_base_ebill_payment_contract.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from datetime import date, timedelta
+
+from odoo.tests.common import Form, SingleTransactionCase
+
+
+class TestCrmOpportunityCurrency(SingleTransactionCase):
```

Comment:
CRM opportunity? :smile: 
Leave it like this... we can change it after the merge.

Reply:
Fixed with https://github.com/OCA/edi/pull/981